NEWS.md

```diff
@@ -1,3 +1,9 @@
+# circacompare 0.1.1.9001
```

Comment:
Could you please add this under the 0.1.1.9000 heading instead of this new version? There hasn't been an associated change to the version number in the DESCRIPTION file to reflect this dev version.

Reply:
All changes now under 0.1.1.9000